packages/react-sdk/src/components/index.ts

```diff
@@ -58,34 +96,24 @@ export const getComponentNames = (): ComponentName[] => {
   return [...uniqueNames.values()] as ComponentName[];
 };
 
-export const getComponentMeta = <Name extends ComponentName>(
-  name: Name
-): Omit<typeof components[Name], "Component"> => {
-  if (
-    registeredComponents != null &&
-    name in registeredComponents &&
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    registeredComponents[name]!.meta != null
-  ) {
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    return { ...components[name], ...registeredComponents[name]!.meta };
+export const getComponentMeta = (name: ComponentName): WsComponentMeta => {
+  if (registeredComponentsMeta != null && name in registeredComponentsMeta) {
+    return { ...meta[name], ...registeredComponentsMeta[name] };
   }
 
-  return components[name];
+  return meta[name];
 };
 
-/**
- * @todo Remove Component from meta and read directly (via import),
- * as meta information is not needed during production render.
- **/
-export const getComponent = <Name extends ComponentName>(
-  name: Name
-): typeof components[Name]["Component"] => {
+export const getComponent = (
+  name: ComponentName
+): typeof components[ComponentName] => {
   return registeredComponents != null && name in registeredComponents
-    ? // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-      (registeredComponents[name]!
-        .Component as typeof components[Name]["Component"])
-    : components[name].Component;
+    ? (registeredComponents[name] as typeof components[ComponentName])
+    : components[name];
+};
+
+export const getComponentMetaProps = (name: ComponentName): MetaProps => {
```

Comment:
Better to merge all props if defaultValue non null then prop will be in result. Having that we mostly overwrite components.